node_modules/oae-core/creatediscussion/js/creatediscussion.js

```diff
@@ -104,7 +104,12 @@ define(['jquery', 'oae.core'], function ($, oae) {
                 // Update visibility for discussion
```

Comment:
In the following scenario (FF 40.0), copying and pasting an email address is not working as expected:

I go to upload and enter an email address to share with. Before pressing enter to select the email address, I do Ctrl+A and Ctrl+C to copy the email address. I then go to "Create discussion" and paste the email address in there. The email address appears to be recognised and receives a pill. However, when clicking "Update", the pasted email address is not in the permissions summary:

<img width="565" alt="screen shot 2015-09-08 at 22 28 13" src="https://cloud.githubusercontent.com/assets/109850/9753983/f69cdd10-5678-11e5-8578-18705173c66d.png">